src/Storefront/Resources/views/storefront/page/account/order-history/order-detail-list.html.twig

```diff
@@ -1,11 +1,7 @@
-{% block page_account_order_item_detail_body %}
```

Comment:
I thought this block would be unnecessary because it is directly followed by another block. I can add `{% block page_account_order_item_detail_body %}` again if the changes become non-breaking anymore?

Reply:
Anything that is being removed 'can' be breaking. In case of an block it is definitely since blocks are there to allow external devs to [extend/override them](https://developer.shopware.com/docs/guides/plugins/plugins/storefront/customize-templates#custom-template-content). 
The removal of the div is also potentially breaking. It may e.g. cause problems with css variable scoping and you don't know where and how the files may be used elsewhere (Plugins, Themes).

If you find an way to make this no breaking that would be great, else to remove the div's you will have to work with feature flags. We would then have to test if the removal causes problems somewhere else in the core, mark the div as deprecated and remove it with the next mayor version.

Reply:
It always would be a breaking change. To get rid of the second `div.order-detail-content-body` you have to delete one of them.  
``` html
<div class="order-detail-content">
    ...
    <div class="order-detail-content-body">
        <div class="order-detail-content-body">
            <div class="order-item-detail-list-item order-detail-item-product">
                ...
            </div>
        </div>
    </div>
</div>
``` 
So i have to work with feature flags. Where I get a Feature ID for the Flag?  Whats the correct way?

 in `Resources/views/storefront/page/account/order-history/order-detail-document.html.twig` i would change like that
```html
{# deprecated '@deprecated tag:v6.5.0 - div `.order-detail-content-body` will be removed' #}
{#  @feature-deprecated (flag:FEATURE_NEXT_11111) #}
<div class="order-detail-content-body">
...
```

and in `Resources/views/storefront/page/account/order-history/order-detail-list.html.twig` I have two suggestions:
- Remove `Block` and `div` 
```html 
{# deprecated '@deprecated tag:v6.5.0 - block `page_account_order_item_detail_body` will be removed use `page_account_order_item_detail_line_items` instead' #}
{#  @feature-deprecated (flag:FEATURE_NEXT_11111) #}
{% block page_account_order_item_detail_body %}
    <div class="order-detail-content-body">
    {% block page_account_order_item_detail_line_items %}
```
or
- Change `class` of `div`
```html
{# deprecated '@deprecated tag:v6.5.0 - class `order-detail-content-body` will be removed  use`order-detail-content-list` instead' #}
{#  @feature-deprecated (flag:FEATURE_NEXT_11111) #}
<div class="order-detail-content-body order-detail-content-list">
...
```
Which one do you prefer?

Reply:
Hey @FKrohmer, sorry that it took so long, somehow my last answer didn't get send.
You can find Information regarding the Compatibility [here](https://developer.shopware.com/docs/resources/guidelines/code/backward-compatibility), which does also link to Information [regarding creating an feature flag](https://github.com/shopware/platform/blob/trunk/adr/2020-08-10-Feature-flag-system.md).
I would prefer just changing the div, blocks are an entry point for plugins/theme devs.

Should it happen again that you don't get an answer for some time, please ping me/the assignee.